.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @Flaconi/engineering
```

Comment:
I would add DevOps as this is a terraform module

Reply:
I would argue with that. Module is mostly used by engineers and was created for them. We do not use it in provisioning our infrastructure.

Reply:
We should at least add @Flaconi/platform instead of @Flaconi/engineering

Reply:
@cytopia up

Reply:
@snovikov Let's add @Flaconi/platform as well as @Flaconi/devops. They might be using it, but I guess the module maintenance (in case of TF updates and such) is still on us